src/data/data_surveys.tsx

```diff
@@ -0,0 +1,219 @@
+const SURVEYDATA = [
+    {
+        name: 'Sara Hui',
+        fun_fact: 'From an island, not a good swimmer. iykyk.',
+        advice: 'Take your time and don’t rush through classes. It’s more enjoyable if you don’t cram all your core classes in one semester.',
+        fav_project: 'Optical Heartbeat Sensor in 20008. I used in infrared sensor for heartbeat detection, a couple of filters to even the signal out, inverters, and comparators to light up an LED for each heartbeat. It was suuper cool!',
```

Comment:
It looks good, but you feel like you are limited to just {attribute: string value}. You can do something like this
fav_project: {
 name: 'name',
 date: 'date',
 etc...
}

Reply:
That's a good idea. Might need to send out the survey again so people can fill out whatever attributes we choose if we want to go that route